pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -160,5 +173,28 @@ func Pod(source *v1alpha1.CatalogSource, name string, image string, saName strin
 			ServiceAccountName: saName,
 		},
 	}
+
+	// Check if this catalogsource is a default one via annotation and set
+	// priorityclass if its annotation exists
+	if annotations[RedHatCatalogAnnotationKey] == RedHatCatalogAnnotationValue {
```

Comment:
Is there something special about the marketplace-operator managed catalog sources that says we should only apply this priority class label to its catalog sources? Why not just honor this annotation on all catalog sources?

Reply:
The original request was to only add priorityclass to default catsrc only. However, it seems we can open to anyone so I will remove this gating.